components/tekton/pipelines/create-cluster/create-cluster.yaml

```diff
@@ -90,10 +95,10 @@ spec:
             git config --global user.name "redhat-testharness"
             git config --global user.email "redhat-partner-solutions@redhat.com"
             git config --global --add safe.directory /workspace/source
-            git checkout -b test-pipeline
+            git checkout -B "$(params.destination_branch)"
             git add $(params.hub_clusters_path)
             git commit -m "Copy folder from pipeline"
-            git push origin test-pipeline
+            git push -f origin "$(params.destination_branch)"
```

Comment:
I don't think we need to force push

Reply:
so tried it without the -f option initially it errors out

Reply:
I think we can start without force push and add this if we will see a need

Reply:
I think I found the issue you had. You need to set `hub_git_revision` in the pipelinerun to the correct branch, the default is main

Reply:
Agreed you shouldn't need a force push is your just adding something to the head of the git log.